Mage/src/main/java/mage/filter/common/FilterSuspendedCard.java

```diff
@@ -0,0 +1,31 @@
+package mage.filter.common;
+
+import mage.abilities.keyword.SuspendAbility;
+import mage.counters.CounterType;
+import mage.filter.FilterCard;
+import mage.filter.predicate.mageobject.AbilityPredicate;
+
+/**
+ * @author skiwkr
+ */
+public class FilterSuspendedCard extends FilterCard {
+
+    public FilterSuspendedCard() {
```

Comment:
If not explicitly checking exile zone, please add doc comment with rules reference for "suspended card":

> 702.62b. A [card](https://yawgatog.com/resources/magic-rules/#card) is "suspended" if it's in the [exile](https://yawgatog.com/resources/magic-rules/#exile) [zone](https://yawgatog.com/resources/magic-rules/#zone), has [suspend](https://yawgatog.com/resources/magic-rules/#suspend), and has a time [counter](https://yawgatog.com/resources/magic-rules/#counter) on it.